tests/unit/pages/embeds/Plio.spec.js

```diff
@@ -135,197 +145,121 @@ describe("Plio.vue", () => {
     expect(mockPlyrVolumeElement[0].style.display).toBe("none");
   });
 
-  it("session should not be created if unauthenticated when plio request is resolved", async () => {
-    const plioId = "123";
-    jest
-      .spyOn(Plio.methods, "setPlayerAspectRatio")
-      .mockImplementation(() => jest.fn());
-    const createSession = jest.spyOn(Plio.methods, "createSession");
-    mount(Plio, {
-      props: {
-        plioId: plioId,
-      },
-    });
-    await flushPromises();
-
-    // resolve the `GET` request waiting in the queue (for receiving plio details)
-    // using the fake response data
-    mockAxios.mockResponse(
-      clonedeep(global.dummyPublishedPlio),
-      mockAxios.queue()[0]
-    );
+  describe("session creation", () => {
+    it("session should not be created if unauthenticated when plio request is resolved", async () => {
```

Comment:
in the title it says that session should not be created but in the code it checks if the session has been created?